tools/StaticAnalysis/Exceptions/Az.Network/SignatureIssues.csv

```diff
@@ -338,4 +338,5 @@
 "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.SetAzureVirtualNetworkGatewayCommand","Set-AzVirtualNetworkGateway","1","8410","Parameter IpConfigurationBgpPeeringAddresses of cmdlet Set-AzVirtualNetworkGateway does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
 "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NetworkWatcher.NewNetworkWatcherConnectionMonitorProtocolConfigurationObject","New-AzNetworkWatcherConnectionMonitorProtocolConfigurationObject","1","8100","New-AzNetworkWatcherConnectionMonitorProtocolConfigurationObject Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NetworkWatcher.NewNetworkWatcherConnectionMonitorProtocolConfigurationObject","New-AzNetworkWatcherConnectionMonitorProtocolConfigurationObject","1","8510","Cmdlet 'New-AzNetworkWatcherConnectionMonitorProtocolConfigurationObject' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
-"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.RemoveAzureSecurityPartnerProviderCommand","Remove-AzSecurityPartnerProvider","1","8510","Cmdlet 'Remove-AzSecurityPartnerProvider' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
\ No newline at end of file
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.RemoveAzureSecurityPartnerProviderCommand","Remove-AzSecurityPartnerProvider","1","8510","Cmdlet 'Remove-AzSecurityPartnerProvider' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
```

Comment:
Please do not suppress the error. You can set a default parameter for "GetAzureSecurityPartnerProviderCommand".

Reply:
@msJinLei I'm not suppressing the error for GetAzureSecurityPartnerProviderCommand ? I don't alter that file at all in this PR?

Reply:
@msJinLei If you are referring to the ShouldProcess line for New-AzRadiusServer, it is because RadiusServer is a sub-object of gateway's VPN server configuration, simply describing a configuration (ip address, score, etc.). There is no need to implement shouldProcess for this subobject especially since running this cmdlet doesn't call Azure.

Reply:
> @msJinLei I'm not suppressing the error for GetAzureSecurityPartnerProviderCommand ? I don't alter that file at all in this PR?

Default parameter set helps user to choose a parameter set when there is a conflict to bind inputs to existing parameter sets. And so this signature error cannot be suppressed. 

Even if the issue is not caused by your PR, it does blocks your PR from merging. That's why I recommend you fix it in your PR.

Reply:
@msJinLei Ok, but I don't see a "GetAzureSecurityPartnerProviderCommand" error being thrown anywhere? The checks both in this PR and in my local static analysis runs don't show GetAzureSecurityPartnerProviderCommand throwing a static analysis error. The checks are passing all green.

Reply:
> @msJinLei Ok, but I don't see a "GetAzureSecurityPartnerProviderCommand" error being thrown anywhere? The checks both in this PR and in my local static analysis runs don't show GetAzureSecurityPartnerProviderCommand throwing a static analysis error. The checks are passing all green.

You should remove this line.
````

"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.RemoveAzureSecurityPartnerProviderCommand","Remove-AzSecurityPartnerProvider","1","8510","Cmdlet 'Remove-AzSecurityPartnerProvider' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
````

Reply:
@msJinLei  Removing that existing static analysis line will cause checks to fail. RemoveAzureSecurityPartnerProviderCommand is owned by a separate team and I do not know the context of which parameter set to make as default for their powershell. Please ask them to change it.

Reply:
> @msJinLei Removing that existing static analysis line will cause checks to fail. RemoveAzureSecurityPartnerProviderCommand is owned by a separate team and I do not know the context of which parameter set to make as default for their powershell. Please ask them to change it.

OK. Please remove this line in you PR anyway. We will have it fixed and then you can rebase master.

Reply:
> @msJinLei Removing that existing static analysis line will cause checks to fail. RemoveAzureSecurityPartnerProviderCommand is owned by a separate team and I do not know the context of which parameter set to make as default for their powershell. Please ask them to change it.
@henry416 We have fixed the issue in the master, please rebase the latest one.

Reply:
@msJinLei Done.